cmd/kube-apiserver/app/server.go

```diff
@@ -495,6 +495,14 @@ func BuildStorageFactory(s *options.ServerRunOptions) (*serverstorage.DefaultSto
 		storageFactory.SetEtcdLocation(groupResource, servers)
 	}
 
+	transformerOverrides, err := s.Etcd.EncryptionProviderConfig.GetTransformerOverrides()
```

Comment:
@smarterclayton Now the parsing is being done in the startup of the server. Is this what was recommended?